crates/block-producer/src/lib.rs

```diff
@@ -8,7 +8,7 @@ mod block_builder;
 mod domain;
 mod errors;
 mod mempool;
-mod store;
+pub mod store;
```

Comment:
Related to the previous comment: we need this only to get the `StoreClient`, right? If so, there may be a better way of getting it.